source/prefab/Card.h

```diff
@@ -0,0 +1,94 @@
+#ifndef EMP_CARD_INTERFACE_H
+#define EMP_CARD_INTERFACE_H
+
+#include "../web/Div.h"
+#include "../tools/string_utils.h"
+#include "Collapse.h"
+#include "FontAwesomeIcon.h"
+
+namespace emp {
+namespace prefab {
+  /// Use Card class to create Bootstrap style cards.
+  class Card : public web::Div{
+  private:
+    std::string card_base = this->GetID();
+    web::Div card_header{emp::to_string(card_base, "_card_header")}; // all header content will be added here
+    web::Div card_body{emp::to_string(card_base, "_card_body")}; // all body content will be added here
+    // Asssigns classes to card elements for styling
+    void AddBootstrap(){
+      this->SetAttr("class", "card");
+      card_header.SetAttr("class", "card-header");
+      card_body.SetAttr("class", "card-body");
+    }
+
+  public:
+    /// @param state indicate whether card should be STAITC, INIT_OPEN, or INIT_CLOSED (default STAITC)
+    /// @param show_glyphs should toggle icons show in collapsible card header? (default true)
```

Comment:
Third param?

Reply:
changed name to id and added a docstring @param